src/components/containers/visibility.tsx

```diff
@@ -125,6 +126,10 @@ export const UKEU = ({ children }: ResponsiveContainerProps) => (
     <CountryBasedContent country_rule="is_uk_eu">{children}</CountryBasedContent>
 )
 
+export const AFRICA = ({ children }: ResponsiveContainerProps) => (
+    <CountryBasedContent country_rule="is_african_country">{children}</CountryBasedContent>
+)
+
```

Comment:
Are we really need declare this in case when you use new rules `useCountryRule()` by `src/pages/why-choose-us/index.tsx` path ?
This way is deprecated and soon file should be removed at all